base_station/gui/src/components/ERDTask.vue

```diff
@@ -168,12 +154,17 @@ export default {
       // Subscriptions
       [
         {'topic': '/odometry', 'type': 'Odometry'},
-        {'topic': '/ra_position', 'type': 'RAPosition'},
+        {'topic': '/sensors', 'type': 'Sensors'},
+        {'topic': '/temperature', 'type': 'Temperature'},
+        {'topic': '/kill_switch', 'type': 'KillSwitch'},
```

Comment:
yo yo yo what's the kill switch doing here

Reply:
ya

Reply:
lol